sdk/eventhub/event-hubs/src/connectionContext.ts

```diff
@@ -48,6 +51,24 @@ export interface ConnectionContext extends ConnectionContextBase {
    * the underlying amqp connection for the EventHub Client.
    */
   managementSession?: ManagementClient;
+
+  /**
+   * Indicates whether the connection is in the process of closing.
```

Comment:
Might be worth documenting how this is different from connection "disconnecting"

Reply:
Updated docs!